packages/block-library/src/audio/editor.scss

```
@@ -16,4 +16,8 @@
 		margin-top: -9px;
 		margin-left: -9px;
 	}
+
+	.components-placeholder.has-illustration {
```

```suggestion
	.block-editor-media-placeholder.is-small {
```

Selectors with the `.components-` prefix should be avoided where possible.